### PR TITLE
Credits scene

### PR DESCRIPTION
**Pull request by PaTo81013**

### CODE REVIEW

**Review by Horothenic — CHANGES_REQUESTED** [review]
Rejected from video

**Comment by PaTo81013** [comment]
[https://youtube.com/live/JWySrw_oIok](url)

**Comment by PaTo81013** [comment]
(https://youtube.com/live/JWySrw_oIok)

File: Where_Is_My_Wife/Assets/_Main/Scripts/UI/Credits/Credits.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+public class Credits : MonoBehaviour
+{
+    private Vector3 _initialPosition;
+    public float _speed = 250f;
+    private float _destroyY = 1080f;
+    public RectTransform _rectTransform;
+    private bool _isResetting = false;
+  
+
```
Comment (Horothenic) [inline Where_Is_My_Wife/Assets/_Main/Scripts/UI/Credits/Credits.cs:13]: Remove this extra space

File: Where_Is_My_Wife/Assets/_Main/Scripts/UI/Credits/Credits.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    private bool _isResetting = false;
+  
+    
+    void Start()
+    {
+        _rectTransform = GetComponent<RectTransform>();
+        _initialPosition = _rectTransform.anchoredPosition;
+    }
+
+
```
Comment (Horothenic) [inline Where_Is_My_Wife/Assets/_Main/Scripts/UI/Credits/Credits.cs:20]: Remove this extra space

File: Where_Is_My_Wife/Assets/_Main/Scripts/UI/Credits/Credits.cs [hunk 3]
```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Credits : MonoBehaviour
```
Comment (Horothenic) [inline Where_Is_My_Wife/Assets/_Main/Scripts/UI/Credits/Credits.cs:5]: Encapsulate this class in `namespace WhereIsMyWife.UI`

**Review by Horothenic — CHANGES_REQUESTED** [review]

**Review by Horothenic — APPROVED** [review]

**Pull request merged**